README.md

```diff
@@ -7,7 +7,7 @@ This a To-Do List App. In this project, I have added a number of functionality t
 
 ## Built With
 
-- Major languages - <b>HTML</b>, <b>CSS</b> and var s = "JavaScript"; alert(s);
+- Major languages - `HTML`, `CSS` and `JavaScript`
 
 ## Getting Started
 
```

Comment:
- [ ] Please it will be nice to update your `readme file` with steps to clone and run you app locally as this will be of help to non technical users who will like to see how your app is running. Please kindly add explicit steps from cloning your app to running it locally.